mlfinlab/tests/test_hcaa.py

```diff
@@ -232,6 +248,31 @@ def test_hcaa_with_input_as_covariance_matrix(self):
         assert len(weights) == self.data.shape[1]
         np.testing.assert_almost_equal(np.sum(weights), 1)
 
+    def test_hcaa_with_input_as_distance_matrix(self):
+        """
+        Test HRP when passing a distance matrix as input.
+        """
+
+        hcaa = HierarchicalClusteringAssetAllocation()
+        returns = ReturnsEstimation().calculate_returns(asset_prices=self.data)
+        covariance = returns.cov()
+        d_matrix = np.zeros_like(covariance)
```

Comment:
Let's use here distance matrix of some_distance(information_variation(percent changes))